content/developer/reference/user_interface/views/include/form.rst

```diff
@@ -0,0 +1,778 @@
+.. _reference/user_interface/views/form:
+
+Form
+====
+
+Form views are used to display the data from a single record. Their root element is
+``<form>``. They are composed of regular HTML_ with additional structural and semantic
+components.
+
+.. code-block:: xml
+
+  <form>
+    ...
+  </form>
+
+Optional attributes_ are added on root element ``<form>`` to customize the view.
+
+:string:
+  string_
+
+  View name
+
+:create:
+  boolean_
+
+  Disable/enable record creation on the view.
+
+:edit:
+  boolean_
+
+  Disable/enable record editing on the view.
+
+:duplicate:
+  boolean_
+
+  Disable/enable record duplication on the view through the **Action** dropdown.
+
+:delete:
+  boolean_
+
+  Disable/enable record deletion on the view through the **Action** dropdown.
+
+:js_class:
+  string_
+
+  Name of the javascript class to use when instantiating the view.
+
+:disable_autofocus:
+  boolean_
+
+  Disable automatic focus of the first field in the view.
+
+.. _reference/user_interface/views/form/semantic:
+
+Semantic components
+-------------------
+
+Semantic components tie into and allow interaction with the Odoo system.
+Placeholders are denoted in all caps.
+
+.. _reference/user_interface/views/form/field:
+
+<field>: render formatted values
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: xml
+
+  <form>
+    <field name="FIELD_NAME"/>
+  </form>
+
+renders (and allow editing of, possibly) a single field of the current
+record. Using several times a field in a form view is supported and the fields
+can receive different values for modifiers 'invisible' and 'readonly'. However,
+the behavior is not guaranteed when several fields exist with different values
+for modifier 'required'. ``<field>`` can have the following attributes:
+
+:name:
+  string_ :ref:`model <reference/orm/model>` field name (mandatory)
+
+  the name of the field to render
+
+:id:
+  string_
+
+  the node id. Useful when there are several occurrences of the same field in
+  the view (see ``label`` component below). Default is the field name.
+
+:widget:
+  string_
+
+  fields have a default rendering based on their type
+  (e.g. :class:`~odoo.fields.Char`, :class:`~odoo.fields.Many2one`).
+
+  The ``widget`` attributes allows using a different rendering method and context.
+  See more information in :ref:`reference/js/widgets`
+
+  .. code-block:: xml
+
+    <field name="tag_ids" widget="many2many_tags"/>
+
+:options:
+  :ref:`python expression <user_interface/views/python_expression>` that defines a dict_
+
+  JSON object specifying configuration option for the field's widget
+  (including default widgets)
+
+  .. code-block:: xml
+
+    <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'FIELD_NAME', 'no_quick_create': True}"/>
+
+:groups:
+  `Comma-separated values`_ whose choices are the :class:`~odoo.addons.base.models.res_users.Groups` reference
+
+  only displays the field for specific users
```

Comment:
could you add/link to the syntax of the groups: what does ! does, how to find the actual string that represent the group (AFAIR it's not the name but the xmlid) etc. pls :)